### ISSUE TITLE
read only json

### ISSUE THREAD

**Issue by twistsmyth** [HelloZeroNet/ZeroNet#1860]
im trying to create content on the users machine that i would have readonly access to  and then reference it as i need it, maybe using their ID. how would i go about doing this? is there something in place or would i need to create a plugin ? , thank you and great work

**Comment by Thunder33345** [comment]
you can just pay the site's address

**Comment by tangdou1** [comment]
No, you can modify the default the pay address (site's address)

**Comment by Thunder33345** [comment]
yes you can also modify and include notes, as there was an extension for the sidebar feature
or paying directly to site's address

**Comment by twistsmyth** [comment]
im not sure it would be secure protocol to use the same private key for site maintenance as well as a static currency store, particularly if your using a zeroid clone to login or any form of automation( private key would have to sit on the server)
